qa-tests-backend/src/main/groovy/orchestratormanager/Kubernetes.groovy

```diff
@@ -642,29 +643,29 @@ class Kubernetes implements OrchestratorMain {
         k8sJob.spec.template.spec.restartPolicy = "Never"
 
         try {
-            println "Told the orchestrator to create job " + job.getName()
+            log.debug "Told the orchestrator to create job " + job.getName()
             return this.jobs.inNamespace(job.namespace).createOrReplace(k8sJob)
         } catch (Exception e) {
-            println "Error creating k8s job" + e
+            log.error("Error creating k8s job", e)
```

Comment:
```suggestion
            log.warn("Error creating k8s job", e)
```
I suggest to stay consistent and use `warn` when exception is handled (maybe it shouldn't but that's a question for another PR).